Core/Core/Extensions/UIViewExtensions.swift

```diff
@@ -126,4 +126,17 @@ extension UIView {
             pin(inside: superview, leading: nil, trailing: nil, top: 0, bottom: 0)
         }
     }
+
+    public func disableIfOnline() {
+        self.alpha = 0.5
```

Comment:
We used 0.3 opacity in the swiftui view, shouldn't we use here the same?

Reply:
yes